src/routes/registerToken.tsx

```diff
@@ -52,13 +52,20 @@ export default class RegisterToken extends React.Component<RegisterTokenProps> {
     return (
       <AppLayout section="registerToken">
         <h1 style={{ marginBottom: 16, marginTop: 32 }}>Registered tokens:</h1>
-        <List
-          itemLayout="vertical"
-          size="small"
-          className="tokens-list"
-          dataSource={tokensStore.ready ? tokensStore.list : undefined}
-          renderItem={item => <Item item={item} />}
-        />
+        {tokensStore.ready ? (
+          <List
+            itemLayout="vertical"
+            size="small"
+            className="tokens-list"
+            dataSource={tokensStore.ready ? tokensStore.list : undefined}
+            renderItem={item => <Item item={item} />}
+          />
+        ) : (
+          <div className="tokens-loading">
```

Comment:
I would recommend to at least swap `span` and `Spin` component. In best case - to make it look like [here](https://ant.design/components/spin/)

Reply:
Alright, I'll fix this

Reply:
I've checked the link, not sure what exactly you are looking out for here. I implemented the Spin

Reply:
![image](https://user-images.githubusercontent.com/3136647/71516634-20a68200-28bb-11ea-875c-edb977a5c1aa.png)
When I said swap `span` and `Spin` I've meant to make it look like this 👆